.travis.yml

```diff
@@ -17,6 +17,7 @@ python:
   - "2.7"
   - "3.4"
   - "3.5"
+  - "3.6"
```

Comment:
As I had advised earlier, is it possible for the `"3.4` and `"3.5"` to be removed and we just have `"2.7"`and `"3.6"` so that our builds on Travis stop taking years to build?